test/cholesky/lkjcholesky.jl

```diff
@@ -181,6 +181,7 @@ using FiniteDifferences
         nkstests = 4 # use for appropriate Bonferroni correction for KS test
 
         @testset "rand" begin
+            @test only(rand(LKJCholesky(1, 0.5)).L) == 1.0
```

Comment:
`only` was introduced in 1.4 IIRC.
```suggestion
            @test rand(LKJCholesky(1, 0.5)) == ones(1, 1)
```